stdlib/REPL/test/repl.jl

```diff
@@ -161,25 +161,27 @@ fake_repl() do stdin_write, stdout_read, repl
     end
 
     # issue #27293
-    let s, old_stdout = stdout
-        write(stdin_write, ";")
-        readuntil(stdout_read, "shell> ")
-        write(stdin_write, "echo ~")
-        s = readuntil(stdout_read, "~")
-
-        proc_stdout_read, proc_stdout = redirect_stdout()
-        get_stdout = @async read(proc_stdout_read, String)
-        try
-            write(stdin_write, "\n")
-            readuntil(stdout_read, "\n")
-            s = readuntil(stdout_read, "\n")
-        finally
-            redirect_stdout(old_stdout)
+    if Sys.isunix()
```

Comment:
Is there a reason you're using `!Sys.iswindows()` in one place and `Sys.isunix()` in the other?

Reply:
This test also needs the unix `echo` binary, the other test only needs `!windows`